archon/infrastructure/supabase/site_pages_repository.py

```
@@ -0,0 +1,307 @@
+            result = self.client.from_(self.table_name).select("*").eq("id", id).execute()
+
+            if not result.data:
+                logger.debug(f"get_by_id(id={id}) -> None")
+                return None
+
+            page = dict_to_site_page(result.data[0])
+            logger.info(f"get_by_id(id={id}) -> found page with url={page.url}")
+            return page
+
+        except Exception as e:
+            logger.error(f"get_by_id(id={id}) -> ERROR: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)